### ISSUE TITLE
Popup Layout Seems To Be Missing Formatting 

### ISSUE THREAD

**Issue by roderik-maker** [thomasloven/hass-browser_mod#597]
**Home Assistant** 2023.6.3
Operating System 10.3
Frontend 20230608.0 - latest

**What I am doing:**
Creating a simple popup using browser_mod. 
```
show_name: true
show_icon: true
type: button
tap_action:
  action: fire-dom-event
  browser_mod:
    service: browser_mod.popup
    data:
      title: The title
      content: The content
      right_button: Right button
      left_button: Left button
```
But also when I use a custom:popup-card 
```
type: custom:popup-card
entity: update.home_assistant_supervisor_update
title: Custom popup
dismissable: true
card:
  type: markdown
  content: This replaces the more-info dialog
right_button: right
left_button: left
```

**What I expected to happen:**
A popup card that looks something like this w.r.t. layout:
<img width="490" alt="Screenshot 2023-07-05 at 18 10 39" src="https://github.com/thomasloven/hass-browser_mod/assets/69784744/9ddf22a2-5f15-4e6c-863c-45ae30e19120">

**What happened instead:**
<img width="657" alt="Screenshot 2023-07-05 at 18 12 38" src="https://github.com/thomasloven/hass-browser_mod/assets/69784744/0dba8874-e3ba-46d3-9d5b-fb43fbc62c7f">

<img width="665" alt="Screenshot 2023-07-05 at 18 12 47" src="https://github.com/thomasloven/hass-browser_mod/assets/69784744/f599ffa3-7121-4ae5-bd25-c71946585806">

**Minimal** steps to reproduce:
See code above.

**Error messages from the browser console:**

```
Navigated to http://homeassistant.local:8123/XXXXX/m
DevTools failed to load source map: Could not load content for chrome-extension://hghakoefmnkhamdhenpbogkeopjlkpoa/contentScript.bundle.js.map: System error: net::ERR_BLOCKED_BY_CLIENT
card-mod.js:5 CARD-MOD 3.2.2 IS INSTALLED
card-mod.js:5 You may not be getting optimal performance out of card-mod.
See https://github.com/thomasloven/lovelace-card-mod#performance-improvements
browser_mod.js:407 BROWSER_MOD 2.2.2 IS INSTALLED    BrowserID: b6dbef99-ffebb09d
state-switch.js:139 STATE-SWITCH 1.9.5 IS INSTALLED 
card-mod.js?hacstag=190927524322:5 You may not be getting optimal performance out of card-mod.
See https://github.com/thomasloven/lovelace-card-mod#performance-improvements
common-translation.ts:16 Fetch finished loading: GET "http://homeassistant.local:8123/static/translations/en-GB-ea496bc7ec14c05a8b05297b14da4615.json".
(anonymous) @ common-translation.ts:16
u @ common-translation.ts:154
(anonymous) @ translations-mixin.ts:469
(anonymous) @ app.ts:11
(anonymous) @ app.ts:11
common-translation.ts:16 Fetch finished loading: GET "http://homeassistant.local:8123/static/translations/lovelace/en-GB-ea496bc7ec14c05a8b05297b14da4615.json".
(anonymous) @ common-translation.ts:16
u @ common-translation.ts:154
_loadFragmentTranslations @ translations-mixin.ts:370
updated @ translations-mixin.ts:108
value @ home-assistant.ts:33
_$AE @ reactive-element.ts:1380
performUpdate @ reactive-element.ts:1345
scheduleUpdate @ reactive-element.ts:1263
_$Ej @ reactive-element.ts:1235
await in _$Ej (async)
requestUpdate @ reactive-element.ts:1210
set @ reactive-element.ts:726
value @ hass-base-mixin.ts:56
_updateHass @ context-mixin.ts:95
(anonymous) @ connection-mixin.ts:239
s @ store.js:19
(anonymous) @ collection.js:25
Promise.then (async)
c @ collection.js:25
d @ collection.js:27
(anonymous) @ collection.js:52
subscribe @ collection.js:95
S @ config.js:16
(anonymous) @ core.ts:118
Promise.then (async)
(anonymous) @ core.ts:113
(anonymous) @ core-FR4zZmgxpbo.js:1
(anonymous) @ core-FR4zZmgxpbo.js:1
locale-data-polyfill.ts:50 Fetch finished loading: GET "http://homeassistant.local:8123/static/locale-data/intl-displaynames/en-GB.json".
o @ locale-data-polyfill.ts:50
(anonymous) @ intl-polyfill.ts:37
Promise.then (async)
t @ intl-polyfill.ts:35
(anonymous) @ intl-polyfill.ts:41
d.a @ async module:49
4631 @ 54625-6TMun0VREMs.js:1
d @ bootstrap:19
Promise.then (async)
(anonymous) @ localize.ts:89
_updateResources @ translations-mixin.ts:426
await in _updateResources (async)
_loadCoreTranslations @ translations-mixin.ts:383
await in _loadCoreTranslations (async)
firstUpdated @ translations-mixin.ts:95
firstUpdated @ themes-mixin.ts:28
firstUpdated @ auth-mixin.ts:16
firstUpdated @ quick-bar-mixin.ts:25
value @ home-assistant.ts:33
_$AE @ reactive-element.ts:1378
performUpdate @ reactive-element.ts:1345
scheduleUpdate @ reactive-element.ts:1263
_$Ej @ reactive-element.ts:1235
await in _$Ej (async)
requestUpdate @ reactive-element.ts:1210
u @ reactive-element.ts:948
b @ reactive-element.ts:931
k @ lit-element.ts:115
F @ hass-base-mixin.ts:6
(anonymous) @ context-mixin.ts:22
(anonymous) @ logging-mixin.ts:23
(anonymous) @ panel-title-mixin.ts:12
(anonymous) @ haptic-mixin.ts:38
(anonymous) @ url-sync-mixin.ts:29
(anonymous) @ dialog-manager-mixin.ts:28
(anonymous) @ notification-mixin.ts:5
(anonymous) @ connection-mixin.ts:39
(anonymous) @ disconnect-toast-mixin.ts:17
(anonymous) @ sidebar-mixin.ts:28
(anonymous) @ more-info-mixin.ts:16
(anonymous) @ translations-mixin.ts:67
(anonymous) @ themes-mixin.ts:24
(anonymous) @ auth-mixin.ts:14
zt @ hass-element.ts:21
(anonymous) @ quick-bar-mixin.ts:23
d @ home-assistant.ts:47
E @ scoped-custom-element-registry.js:385
t @ scoped-custom-element-registry.js:236
window.CustomElementRegistry.define @ scoped-custom-element-registry.js:85
finisher @ custom-element.ts:41
runClassFinishers @ decorate.js:237
s @ decorate.js:15
(anonymous) @ home-assistant.ts:33
(anonymous) @ app.ts:11
(anonymous) @ app.ts:11
apexcharts-card.js?hacstag=331701152204:835  APEXCHARTS-CARD  v2.0.4 
simple-thermostat.js:292 simple-thermostat: 2.5.0
mushroom.js?hacstag=444350375281:3101 🍄 Mushroom 🍄 - 2.8.1
button-card.js?hacstag=146194325350:425   BUTTON-CARD   Version 3.5.0 
vacuum-card.js?hacstag=261291295273:4  VACUUM-CARD  2.7.2
bar-card.js?hacstag=163363577320:4855   BAR-CARD   Version 3.1.7    
vertical-stack-in-card.js?hacstag=142051833044:1 vertical-stack-in-cardVersion: 0.4.4
mini-graph-card-bundle.js?hacstag=1512800620110:1  MINI-GRAPH-CARD  0.11.0 
restriction-card.js?hacstag=215633404129:200   RESTRICTION-CARD    Version 1.2.9     
```

---

**By replacing the space in the checkboxes ([ ]) with an X below, I indicate that I:**

- [X] Understand that this is a channel for reporting bugs, not a support forum (https://community.home-assistant.io/).

- [X] Have made sure I am using the latest version of the plugin.

- [X] Have followed the troubleshooting steps of the "Common Problems" section of https://github.com/thomasloven/hass-config/wiki/Lovelace-Plugins.

- [X] Understand that leaving one or more boxes unticked or failure to follow the template above may increase the time required to handle my bug-report, or cause it to be closed without further action.

**Comment by VDRainer** [comment]
Not sure if i missed this in HA 2023.6, but i'm seeing this in 2023.7 beta.

**Comment by moonter45** [comment]
Check this: https://github.com/thomasloven/hass-browser_mod/issues/587
There is a temporary workaround.

**Comment by thomasloven** [comment]
Fixed by #588
